test/typings/graphql.test-d.ts

```diff
@@ -1,193 +1,211 @@
 import { parse } from 'graphql'
 import { graphql, HttpResponse, passthrough } from 'msw'
+import { describe, expectTypeOf, it } from 'vitest'
+
+describe('GraphQL handlers', () => {
```

Comment:
Let's keep our type tests flat as we do out regular tests. Drop the `describe()` and make the test names reflect what they are testing. 

I don't mind mixing a bunch of behaviors in type tests. The price of maintaining them is rather low.